src/internal/models/IVolumeActionModel.ts

```diff
@@ -0,0 +1,17 @@
+import {IActionModel} from "./IActionModel.js";
+
+export interface IVolumeByPercent{
+  percent: number;
+}
+
+export interface IVolumeByDecibels{
+  decibels: number;
+}
+
+export type IVolumeMute = 'mute';
+
+export type IVolumeValue = IVolumeByDecibels | IVolumeByPercent | IVolumeMute;
```

Comment:
@patrick-tolosa you think this can be changed to:
IVolumeMute {
  mode: ‘mute’
}
IVolumteByPercent {
mode: ‘percent’;
value: number;
}
IVolumeByDecibels{
mode: ‘decibels’;
value: number;
}